lesson1/lesson1Tasks.py

```diff
@@ -0,0 +1,33 @@
+if __name__ == '__main__':
+    # Посчитайте значение выражения:3^128*2^128
+    result = 3 ^ 128 * 2 ^ 128
+    print(result)
+
+    # Выведете слово python 100 раз
+    print('python' * 100)
+
+    # Напишите программу, которая считает сколько букв ‘o’ в заданной строке
+    strToCount = 'ololololololololololololololololo'
+    print(strToCount.count('o'))
+
+    # Напишите программу, которая сортирует заданный список из чисел
+    splittedList = list('34358798475984603620369236802938602398')
+    splittedList.sort()
+    print(splittedList)
+
+    # Напишите программу, которая считает сумму последних цифр двух введенных пользователем чисел
+    firstInput = input()
+    secondInput = input()
+    firstDigit = int(list(firstInput)[list(firstInput).__len__() - 1])
```

Comment:
Never call private methods like __len__
```suggestion
    len(firstInput)
```